### DISCUSSION TITLE
Can StylePile be used with deforum?

### DISCUSSION THREAD

**Discussion by sunilw** [some9000/StylePile#47] [Q&A]
I've been very much enjoying StylePile.
I've been using it to direct the aesthetic quality of images and I've added some styles in my own local install.

Recently I've been experimenting with deforum.
I'd like to know if it's possible to use StylePile to influence the images in deforum animations.

**Comment by some9000** [comment]
Sorry for the mega delayed reply. Was away from coding for a while. To be completely honest I had no idea what deforum is. But, at the end of the day, StylePile just lets you do all sorts of things with prompts. Trying to implement cooperation with other SD elements or such is waaay out of my skill level.

Do you mind sharing what styles you added?

**Reply by sunilw** [reply]
Since the time that I made this request I've learnt more about both StylePile and Deforum.

I think my request was naive.

What would be really though, would be a kind of prompt building script for deforum in Automatic1111. And it would it be cool if that had a StylePile-like functionality.

But that would be a pretty sophisticated feature, and difficult to integrate into gradio - which looks pretty clunky to work with.

The Automatic1111 interface into Deforum isn't pretty good in its current state, all things considered.

At the current rate of change across technologies, I suspect that by the time an "advanced prompt builder for Deforum" might be practical, Deforum, automatic1111 and maybe even stable diffusion might be different beasts.

**Comment by some9000** [comment]
I see. And it's true, this technology just keeps flying forward with lighting speed.

Anyway, if you'd like you can check out the development version of the next step for StylePile: https://github.com/some9000/StylePile/tree/Error-hunting
